src/ansys/aedt/core/modeler/cad/modeler.py

```diff
@@ -833,12 +834,13 @@ def change_cs_mode(self, mode_type=0):
             ``True`` when successful, ``False`` when failed.
 
         """
-        legacy_update = self.auto_update
+        previous_auto_update = self.auto_update
         self.auto_update = False
         if mode_type == 0:  # "Axis/Position"
             if self.props and (self.props["Mode"] == "Euler Angle ZXZ" or self.props["Mode"] == "Euler Angle ZYZ"):
                 self.props["Mode"] = "Axis/Position"
-                x, y, z = GeometryOperators.quaternion_to_axis(self.quaternion)
+                m = self.quaternion.to_rotation_matrix()
+                x, y, z = Quaternion.rotation_matrix_to_axis(m)
```

Comment:
```suggestion
                x, y, _ = Quaternion.rotation_matrix_to_axis(m)
```

This should resolve the codacity failure.